[2024-03-18T02:18:26.617+0000] {processor.py:153} INFO - Started process (PID=65) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:18:26.618+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:18:26.618+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:18:26.618+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:18:26.631+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:18:26.630+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:18:26.632+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:18:26.646+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.031 seconds
[2024-03-18T02:18:56.809+0000] {processor.py:153} INFO - Started process (PID=101) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:18:56.811+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:18:56.811+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:18:56.811+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:18:56.818+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:18:56.818+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:18:56.819+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:18:56.835+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.030 seconds
[2024-03-18T02:19:27.044+0000] {processor.py:153} INFO - Started process (PID=154) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:19:27.044+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:19:27.045+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:19:27.045+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:19:27.051+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:19:27.050+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:19:27.051+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:19:27.063+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.021 seconds
[2024-03-18T02:19:57.269+0000] {processor.py:153} INFO - Started process (PID=200) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:19:57.269+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:19:57.269+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:19:57.269+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:19:57.274+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:19:57.274+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:19:57.275+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:19:57.297+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.030 seconds
[2024-03-18T02:20:27.493+0000] {processor.py:153} INFO - Started process (PID=246) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:20:27.494+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:20:27.495+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:20:27.495+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:20:27.502+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:20:27.501+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:20:27.502+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:20:27.526+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.035 seconds
[2024-03-18T02:20:57.724+0000] {processor.py:153} INFO - Started process (PID=301) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:20:57.725+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:20:57.726+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:20:57.726+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:20:57.735+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:20:57.734+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:20:57.735+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:20:57.761+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.041 seconds
[2024-03-18T02:21:27.931+0000] {processor.py:153} INFO - Started process (PID=338) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:21:27.932+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:21:27.932+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:21:27.932+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:21:27.938+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:21:27.937+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:21:27.938+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:21:27.952+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.025 seconds
[2024-03-18T02:21:58.126+0000] {processor.py:153} INFO - Started process (PID=392) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:21:58.128+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:21:58.129+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:21:58.129+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:21:58.139+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:21:58.138+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:21:58.140+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:21:58.170+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.049 seconds
[2024-03-18T02:22:28.353+0000] {processor.py:153} INFO - Started process (PID=429) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:22:28.353+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:22:28.354+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:22:28.354+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:22:28.358+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:22:28.358+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:22:28.359+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:22:28.381+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.031 seconds
[2024-03-18T02:22:58.598+0000] {processor.py:153} INFO - Started process (PID=484) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:22:58.599+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:22:58.600+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:22:58.600+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:22:58.613+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:22:58.611+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:22:58.614+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:22:58.650+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.056 seconds
[2024-03-18T02:23:28.848+0000] {processor.py:153} INFO - Started process (PID=539) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:23:28.849+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:23:28.849+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:23:28.849+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:23:28.855+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:23:28.854+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:23:28.855+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:23:28.877+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.031 seconds
[2024-03-18T02:23:59.068+0000] {processor.py:153} INFO - Started process (PID=576) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:23:59.069+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:23:59.070+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:23:59.070+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:23:59.081+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:23:59.080+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:23:59.081+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:23:59.114+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.050 seconds
[2024-03-18T02:24:29.303+0000] {processor.py:153} INFO - Started process (PID=630) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:24:29.304+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:24:29.304+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:24:29.304+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:24:29.313+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:24:29.312+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:24:29.314+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:24:29.338+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T02:24:59.522+0000] {processor.py:153} INFO - Started process (PID=676) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:24:59.523+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:24:59.524+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:24:59.524+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:24:59.531+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:24:59.530+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:24:59.531+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:24:59.557+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T02:25:29.743+0000] {processor.py:153} INFO - Started process (PID=722) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:25:29.743+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:25:29.745+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:25:29.745+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:25:29.755+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:25:29.754+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:25:29.756+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:25:29.779+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.039 seconds
[2024-03-18T02:25:59.931+0000] {processor.py:153} INFO - Started process (PID=777) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:25:59.932+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:25:59.933+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:25:59.933+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:25:59.942+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:25:59.941+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:25:59.942+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:25:59.965+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.039 seconds
[2024-03-18T02:26:30.164+0000] {processor.py:153} INFO - Started process (PID=813) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:26:30.165+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:26:30.166+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:26:30.166+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:26:30.174+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:26:30.174+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:26:30.175+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:26:30.198+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T02:27:00.383+0000] {processor.py:153} INFO - Started process (PID=868) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:27:00.384+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:27:00.385+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:27:00.385+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:27:00.393+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:27:00.392+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:27:00.393+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:27:00.417+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.039 seconds
[2024-03-18T02:27:30.598+0000] {processor.py:153} INFO - Started process (PID=922) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:27:30.599+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:27:30.599+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:27:30.599+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:27:30.604+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:27:30.604+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:27:30.605+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:27:30.620+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.025 seconds
[2024-03-18T02:28:00.794+0000] {processor.py:153} INFO - Started process (PID=959) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:28:00.795+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:28:00.795+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:28:00.795+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:28:00.800+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:28:00.799+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:28:00.800+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:28:00.827+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.036 seconds
[2024-03-18T02:28:30.992+0000] {processor.py:153} INFO - Started process (PID=1013) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:28:30.992+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:28:30.993+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:28:30.993+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:28:30.999+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:28:30.998+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:28:30.999+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:28:31.024+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.036 seconds
[2024-03-18T02:29:01.202+0000] {processor.py:153} INFO - Started process (PID=1068) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:29:01.203+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:29:01.204+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:29:01.203+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:29:01.210+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:29:01.209+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:29:01.211+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:29:01.237+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T02:29:31.427+0000] {processor.py:153} INFO - Started process (PID=1105) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:29:31.428+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:29:31.429+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:29:31.429+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:29:31.434+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:29:31.434+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:29:31.435+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:29:31.458+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.035 seconds
[2024-03-18T02:30:01.641+0000] {processor.py:153} INFO - Started process (PID=1160) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:30:01.642+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:30:01.643+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:30:01.643+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:30:01.651+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:30:01.651+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:30:01.652+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:30:01.678+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.039 seconds
[2024-03-18T02:30:31.854+0000] {processor.py:153} INFO - Started process (PID=1197) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:30:31.855+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:30:31.856+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:30:31.855+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:30:31.865+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:30:31.864+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:30:31.865+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:30:31.893+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.044 seconds
[2024-03-18T02:31:02.083+0000] {processor.py:153} INFO - Started process (PID=1251) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:31:02.084+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:31:02.084+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:31:02.084+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:31:02.094+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:31:02.093+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:31:02.095+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:31:02.117+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T02:31:32.273+0000] {processor.py:153} INFO - Started process (PID=1305) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:31:32.274+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:31:32.275+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:31:32.275+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:31:32.285+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:31:32.283+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:31:32.286+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:31:32.304+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.034 seconds
[2024-03-18T02:32:02.481+0000] {processor.py:153} INFO - Started process (PID=1341) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:32:02.481+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:32:02.482+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:32:02.482+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:32:02.487+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:32:02.486+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:32:02.487+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:32:02.512+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.034 seconds
[2024-03-18T02:32:32.701+0000] {processor.py:153} INFO - Started process (PID=1395) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:32:32.701+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:32:32.702+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:32:32.702+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:32:32.709+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:32:32.708+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:32:32.710+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:32:32.736+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T02:33:02.892+0000] {processor.py:153} INFO - Started process (PID=1439) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:33:02.892+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:33:02.893+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:33:02.892+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:33:02.897+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:33:02.897+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:33:02.898+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:33:02.923+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.034 seconds
[2024-03-18T02:33:33.120+0000] {processor.py:153} INFO - Started process (PID=1487) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:33:33.121+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:33:33.121+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:33:33.121+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:33:33.126+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:33:33.125+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:33:33.126+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:33:33.150+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.032 seconds
[2024-03-18T02:34:03.334+0000] {processor.py:153} INFO - Started process (PID=1542) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:34:03.334+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:34:03.335+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:34:03.335+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:34:03.343+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:34:03.342+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:34:03.343+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:34:03.364+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.032 seconds
[2024-03-18T02:34:33.522+0000] {processor.py:153} INFO - Started process (PID=1578) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:34:33.523+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:34:33.524+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:34:33.524+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:34:33.533+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:34:33.531+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:34:33.534+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:34:33.554+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.035 seconds
[2024-03-18T02:35:03.722+0000] {processor.py:153} INFO - Started process (PID=1633) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:35:03.722+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:35:03.724+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:35:03.723+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:35:03.735+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:35:03.733+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:35:03.736+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:35:03.765+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.047 seconds
[2024-03-18T02:36:14.527+0000] {processor.py:153} INFO - Started process (PID=84) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:36:14.528+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:36:14.529+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:36:14.529+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:36:14.537+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:36:14.536+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:36:14.537+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:36:14.551+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.025 seconds
[2024-03-18T02:36:44.788+0000] {processor.py:153} INFO - Started process (PID=121) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:36:44.789+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:36:44.790+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:36:44.789+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:36:44.804+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:36:44.801+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:36:44.805+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:36:44.835+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.052 seconds
[2024-03-18T02:37:15.034+0000] {processor.py:153} INFO - Started process (PID=175) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:37:15.034+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:37:15.035+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:37:15.035+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:37:15.042+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:37:15.041+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:37:15.043+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:37:15.070+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.039 seconds
[2024-03-18T02:37:45.239+0000] {processor.py:153} INFO - Started process (PID=220) to work on /opt/airflow/dags/airflow.py
[2024-03-18T02:37:45.246+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T02:37:45.247+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:37:45.247+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T02:37:45.261+0000] {logging_mixin.py:137} INFO - [2024-03-18T02:37:45.257+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T02:37:45.262+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T02:37:45.302+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.067 seconds
[2024-03-18T04:04:31.169+0000] {processor.py:153} INFO - Started process (PID=40) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:04:31.170+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:04:31.171+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:04:31.171+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:04:31.183+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:04:31.182+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:04:31.184+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:04:31.197+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.030 seconds
[2024-03-18T04:05:01.394+0000] {processor.py:153} INFO - Started process (PID=95) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:05:01.394+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:05:01.395+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:05:01.395+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:05:01.401+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:05:01.400+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:05:01.402+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:05:01.427+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.036 seconds
[2024-03-18T04:05:31.642+0000] {processor.py:153} INFO - Started process (PID=132) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:05:31.642+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:05:31.643+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:05:31.643+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:05:31.653+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:05:31.651+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:05:31.654+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:05:31.677+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T04:05:56.813+0000] {processor.py:153} INFO - Started process (PID=169) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:05:56.814+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:05:56.814+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:05:56.814+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:05:56.821+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:05:56.820+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:05:56.821+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:05:56.838+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.027 seconds
[2024-03-18T04:06:27.072+0000] {processor.py:153} INFO - Started process (PID=224) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:06:27.073+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:06:27.074+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:06:27.074+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:06:27.087+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:06:27.086+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:06:27.088+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:06:27.115+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.046 seconds
[2024-03-18T04:06:57.277+0000] {processor.py:153} INFO - Started process (PID=279) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:06:57.278+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:06:57.279+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:06:57.278+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:06:57.284+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:06:57.283+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 5, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:06:57.284+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:06:57.307+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.032 seconds
[2024-03-18T04:07:18.459+0000] {processor.py:153} INFO - Started process (PID=298) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:07:18.460+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:07:18.460+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:07:18.460+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:07:18.462+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:07:18.462+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 8
    from .. /src.Merge_Files import merge_files
            ^
SyntaxError: invalid syntax
[2024-03-18T04:07:18.463+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:07:18.478+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.021 seconds
[2024-03-18T04:07:42.650+0000] {processor.py:153} INFO - Started process (PID=344) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:07:42.651+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:07:42.652+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:07:42.652+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:07:42.656+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:07:42.655+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 8
    from ./src.Merge_Files import merge_files
          ^
SyntaxError: invalid syntax
[2024-03-18T04:07:42.656+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:07:42.676+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.029 seconds
[2024-03-18T04:08:12.853+0000] {processor.py:153} INFO - Started process (PID=389) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:08:12.854+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:08:12.854+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:08:12.854+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:08:12.857+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:08:12.856+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 8
    from ./src.Merge_Files import merge_files
          ^
SyntaxError: invalid syntax
[2024-03-18T04:08:12.857+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:08:12.879+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.030 seconds
[2024-03-18T04:08:43.072+0000] {processor.py:153} INFO - Started process (PID=442) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:08:43.072+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:08:43.073+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:08:43.073+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:08:43.076+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:08:43.076+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 8
    from ./src.Merge_Files import merge_files
          ^
SyntaxError: invalid syntax
[2024-03-18T04:08:43.077+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:08:43.105+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.036 seconds
[2024-03-18T04:09:13.291+0000] {processor.py:153} INFO - Started process (PID=479) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:09:13.292+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:09:13.292+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:09:13.292+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:09:13.298+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:09:13.296+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 8
    from ./src.Merge_Files import merge_files
          ^
SyntaxError: invalid syntax
[2024-03-18T04:09:13.298+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:09:13.320+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.032 seconds
[2024-03-18T04:09:43.487+0000] {processor.py:153} INFO - Started process (PID=534) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:09:43.488+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:09:43.489+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:09:43.489+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:09:43.491+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:09:43.491+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 8
    from ./src.Merge_Files import merge_files
          ^
SyntaxError: invalid syntax
[2024-03-18T04:09:43.492+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:09:43.518+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.033 seconds
[2024-03-18T04:10:13.706+0000] {processor.py:153} INFO - Started process (PID=571) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:10:13.707+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:10:13.708+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:10:13.708+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:10:13.711+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:10:13.710+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 8
    from ./src.Merge_Files import merge_files
          ^
SyntaxError: invalid syntax
[2024-03-18T04:10:13.711+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:10:13.739+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.035 seconds
[2024-03-18T04:10:43.913+0000] {processor.py:153} INFO - Started process (PID=626) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:10:43.914+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:10:43.915+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:10:43.914+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:10:43.917+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:10:43.916+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 8
    from ./src.Merge_Files import merge_files
          ^
SyntaxError: invalid syntax
[2024-03-18T04:10:43.917+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:10:43.945+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.033 seconds
[2024-03-18T04:11:11.132+0000] {processor.py:153} INFO - Started process (PID=663) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:11:11.133+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:11:11.134+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:11:11.134+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:11:11.143+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:11:11.141+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:11:11.143+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:11:11.163+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.035 seconds
[2024-03-18T04:11:41.334+0000] {processor.py:153} INFO - Started process (PID=717) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:11:41.335+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:11:41.336+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:11:41.336+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:11:41.342+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:11:41.341+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:11:41.342+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:11:41.370+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.039 seconds
[2024-03-18T04:12:11.529+0000] {processor.py:153} INFO - Started process (PID=771) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:12:11.530+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:12:11.530+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:12:11.530+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:12:11.536+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:12:11.535+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:12:11.536+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:12:11.559+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.032 seconds
[2024-03-18T04:12:41.770+0000] {processor.py:153} INFO - Started process (PID=808) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:12:41.771+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:12:41.772+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:12:41.772+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:12:41.782+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:12:41.781+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:12:41.782+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:12:41.801+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.035 seconds
[2024-03-18T04:13:11.995+0000] {processor.py:153} INFO - Started process (PID=863) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:13:11.996+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:13:11.998+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:13:11.997+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:13:12.014+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:13:12.012+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:13:12.015+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:13:12.036+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.047 seconds
[2024-03-18T04:14:10.172+0000] {processor.py:153} INFO - Started process (PID=40) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:14:10.173+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:14:10.173+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:14:10.173+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:14:10.188+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:14:10.186+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:14:10.188+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:14:10.217+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.049 seconds
[2024-03-18T04:14:40.399+0000] {processor.py:153} INFO - Started process (PID=95) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:14:40.400+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:14:40.401+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:14:40.401+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:14:40.407+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:14:40.406+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:14:40.408+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:14:40.424+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.030 seconds
[2024-03-18T04:15:10.649+0000] {processor.py:153} INFO - Started process (PID=132) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:15:10.650+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:15:10.650+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:15:10.650+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:15:10.658+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:15:10.657+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:15:10.658+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:15:10.675+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.028 seconds
[2024-03-18T04:15:40.893+0000] {processor.py:153} INFO - Started process (PID=187) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:15:40.894+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:15:40.895+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:15:40.895+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:15:40.909+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:15:40.907+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:15:40.910+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:15:40.951+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.061 seconds
[2024-03-18T04:16:11.126+0000] {processor.py:153} INFO - Started process (PID=242) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:16:11.127+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:16:11.127+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:16:11.127+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:16:11.133+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:16:11.132+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:16:11.133+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:16:11.164+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.042 seconds
[2024-03-18T04:16:41.351+0000] {processor.py:153} INFO - Started process (PID=278) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:16:41.352+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:16:41.353+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:16:41.352+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:16:41.365+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:16:41.362+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:16:41.365+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:16:41.403+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.058 seconds
[2024-03-18T04:17:11.570+0000] {processor.py:153} INFO - Started process (PID=333) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:17:11.571+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:17:11.573+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:17:11.572+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:17:11.584+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:17:11.583+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:17:11.585+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:17:11.604+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T04:17:41.812+0000] {processor.py:153} INFO - Started process (PID=370) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:17:41.812+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:17:41.813+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:17:41.813+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:17:41.826+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:17:41.822+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:17:41.827+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:17:41.869+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.061 seconds
[2024-03-18T04:18:12.037+0000] {processor.py:153} INFO - Started process (PID=425) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:18:12.038+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:18:12.039+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:18:12.038+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:18:12.045+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:18:12.045+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:18:12.046+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:18:12.072+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T04:18:42.238+0000] {processor.py:153} INFO - Started process (PID=480) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:18:42.239+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:18:42.239+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:18:42.239+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:18:42.245+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:18:42.244+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:18:42.246+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:18:42.270+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.035 seconds
[2024-03-18T04:19:12.484+0000] {processor.py:153} INFO - Started process (PID=517) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:19:12.485+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:19:12.486+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:19:12.486+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:19:12.493+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:19:12.492+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:19:12.493+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:19:12.520+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.040 seconds
[2024-03-18T04:19:42.722+0000] {processor.py:153} INFO - Started process (PID=569) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:19:42.723+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:19:42.723+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:19:42.723+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:19:42.734+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:19:42.732+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:19:42.735+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:19:42.760+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.042 seconds
[2024-03-18T04:20:12.970+0000] {processor.py:153} INFO - Started process (PID=606) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:20:12.970+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:20:12.971+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:20:12.971+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:20:12.977+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:20:12.976+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:20:12.977+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:20:13.017+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.050 seconds
[2024-03-18T04:20:43.190+0000] {processor.py:153} INFO - Started process (PID=661) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:20:43.191+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:20:43.192+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:20:43.192+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:20:43.199+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:20:43.198+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:20:43.199+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:20:43.224+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T04:21:13.411+0000] {processor.py:153} INFO - Started process (PID=716) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:21:13.412+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:21:13.413+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:21:13.413+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:21:13.420+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:21:13.419+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:21:13.420+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:21:13.449+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.040 seconds
[2024-03-18T04:21:43.632+0000] {processor.py:153} INFO - Started process (PID=752) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:21:43.634+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:21:43.636+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:21:43.635+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:21:43.645+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:21:43.644+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:21:43.646+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:21:43.664+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T04:22:13.820+0000] {processor.py:153} INFO - Started process (PID=805) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:22:13.820+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:22:13.821+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:22:13.821+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:22:13.828+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:22:13.827+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:22:13.828+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:22:13.857+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.040 seconds
[2024-03-18T04:22:44.025+0000] {processor.py:153} INFO - Started process (PID=841) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:22:44.026+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:22:44.026+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:22:44.026+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:22:44.031+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:22:44.030+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:22:44.031+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:22:44.055+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.033 seconds
[2024-03-18T04:23:14.250+0000] {processor.py:153} INFO - Started process (PID=896) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:23:14.251+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:23:14.252+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:23:14.252+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:23:14.261+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:23:14.260+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:23:14.262+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:23:14.279+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.032 seconds
[2024-03-18T04:23:44.457+0000] {processor.py:153} INFO - Started process (PID=951) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:23:44.458+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:23:44.458+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:23:44.458+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:23:44.463+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:23:44.462+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:23:44.463+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:23:44.478+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.023 seconds
[2024-03-18T04:24:14.698+0000] {processor.py:153} INFO - Started process (PID=988) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:24:14.699+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:24:14.699+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:24:14.699+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:24:14.704+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:24:14.704+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:24:14.705+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:24:14.729+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.034 seconds
[2024-03-18T04:24:44.910+0000] {processor.py:153} INFO - Started process (PID=1043) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:24:44.910+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:24:44.911+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:24:44.911+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:24:44.918+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:24:44.917+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:24:44.918+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:24:44.947+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.041 seconds
[2024-03-18T04:25:15.134+0000] {processor.py:153} INFO - Started process (PID=1080) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:25:15.135+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:25:15.135+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:25:15.135+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:25:15.140+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:25:15.139+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:25:15.140+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:25:15.166+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.034 seconds
[2024-03-18T04:25:45.360+0000] {processor.py:153} INFO - Started process (PID=1134) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:25:45.361+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:25:45.362+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:25:45.361+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:25:45.367+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:25:45.366+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:25:45.367+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:25:45.395+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T04:26:15.571+0000] {processor.py:153} INFO - Started process (PID=1189) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:26:15.572+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:26:15.572+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:26:15.572+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:26:15.578+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:26:15.577+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:26:15.579+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:26:15.605+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.039 seconds
[2024-03-18T04:26:45.782+0000] {processor.py:153} INFO - Started process (PID=1225) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:26:45.783+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:26:45.784+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:26:45.784+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:26:45.796+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:26:45.795+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:26:45.797+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:26:45.817+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.038 seconds
[2024-03-18T04:27:15.998+0000] {processor.py:153} INFO - Started process (PID=1280) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:27:15.999+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:27:15.999+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:27:15.999+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:27:16.004+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:27:16.003+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:27:16.005+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:27:16.029+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.033 seconds
[2024-03-18T04:27:46.198+0000] {processor.py:153} INFO - Started process (PID=1324) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:27:46.200+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:27:46.200+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:27:46.200+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:27:46.207+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:27:46.206+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:27:46.208+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:27:46.221+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.026 seconds
[2024-03-18T04:28:16.413+0000] {processor.py:153} INFO - Started process (PID=1372) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:28:16.414+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:28:16.415+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:28:16.415+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:28:16.424+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:28:16.422+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:28:16.424+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:28:16.459+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.049 seconds
[2024-03-18T04:28:46.634+0000] {processor.py:153} INFO - Started process (PID=1427) to work on /opt/airflow/dags/airflow.py
[2024-03-18T04:28:46.635+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2024-03-18T04:28:46.637+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:28:46.637+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2024-03-18T04:28:46.651+0000] {logging_mixin.py:137} INFO - [2024-03-18T04:28:46.649+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 10, in <module>
    from src.Convert_json_csv import process_files
ModuleNotFoundError: No module named 'src'
[2024-03-18T04:28:46.652+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2024-03-18T04:28:46.670+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.042 seconds
